.eslintrc.js

```diff
@@ -0,0 +1,64 @@
+module.exports = {
```

Comment:
thanks for getting this wired up! 🎉 

Noticed spinning up on `arches-rdm` that moving the dotfiles up a directory causes `updateproject` to throw like this:

```py
  File "/Users/jwalls/prj/arches/arches/management/commands/updateproject.py", line 35, in update_to_v7
    shutil.copy2(os.path.join(settings.ROOT_DIR, dotfile), settings.APP_ROOT)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/shutil.py", line 463, in copy2
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/shutil.py", line 260, in copyfile
    with open(src, 'rb') as fsrc:
         ^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/jwalls/prj/arches/arches/.eslintrc.js'```

Reply:
Yes, this is still in draft mode and I'm making my final round of checks before putting it out for review